### PR TITLE
Added test for Jira

### PR DESCRIPTION
**Pull request by nuCleaRR**
I think additional improvements are needed:
Add specific IDs to task description, project name and task link fields.
Stop task before test launching and after test running too. In first case we should add mechanism to determine when TT had been finished its loading.

### CODE REVIEW

File: test/webdriverio/customCommands.js [hunk 1]
```diff
@@ -10,10 +10,11 @@ browser.addCommand("login", function(serviceName) {
     .setValue(service.login.usernameField, service.login.username)
     .setValue(service.login.passwordField, service.login.password)
     .click(service.login.submitButton)
+    .pause(1000)
```
Comment (ArtemAvramenko) [inline test/webdriverio/customCommands.js]: Use waitUntil - http://www.webdriver.io/api/utility/waitUntil.html

File: test/webdriverio/specs/jira.spec.js [hunk 2]
```diff
@@ -18,37 +22,56 @@ describe("Jira integration spec", function () {
...
+      .getText('.timer-active div .text-overflow').then(function (text) {
+        expect(text).toBe(taskName);
+      })
+      .getText('.timer-active .timer-td-project').then(function (text) {
+        expect(text).toBe(projectName);
+      })
+      .getAttribute('.timer-active a.flex-item-no-shrink', 'href').then(function (text) {
+        expect(text).toBe(href);
+      })
+      .then(done);
```
Comment (ArtemAvramenko) [inline test/webdriverio/specs/jira.spec.js]: You can return promise instead of done call.

**Comment by ArtemAvramenko** [comment]
We should not add identifiers because multiple tasks are shown on single page.

File: test/webdriverio/customCommands.js (hunk 1 above)
Comment (alexserdyuk) [inline test/webdriverio/customCommands.js]: It's not needed here. We just need to pause and allow time for server to process the request.

**Comment by alexserdyuk** [comment]
Webdriver does not support REST API calls.

**Comment by nuCleaRR** [comment]
> We should not add identifiers because multiple tasks are shown on single page.

I have already added class 'Active' to the table row definition to determine what task is an active now.
For example, now we have such selector for getting task description:
```css
.timer-active div .text-overflow
```
Or even more ugly for the task link:
```css
.timer-active a.flex-item-no-shrink
```
Instead I suggest to add specific IDs for that fields and code will become:
```css
.timer-active #task-description
.timer-active #task-link
```

**Comment by ArtemAvramenko** [comment]
IDs should be unique. We can add specific classes, but not identifiers.

**Pull request merged**